### ISSUE TITLE
add referralCode from Aave to contract

### ISSUE THREAD

**Issue by Deepcryptodive** [Good-Ghosting/goodghosting-smart-contracts#7]
## Original post:
👻 Aave has a referral program, consisting of a 20% fee-share.
Each dApp that calls the methods `deposit()` or `borrow()` gets a slice of their origination fee.
We will call `deposit()` whenever we deposit DAI into aDAI. Hence, we can earn some platform fees.

More info: [https://docs.aave.com/developers/integrating-aave/referral-program](https://docs.aave.com/developers/integrating-aave/referral-program)

> Aave has a 20% fee-share referral program. Aave loans have a ~~0.25%~~ origination fee on the loan amount. 80% of this fee is used to buy on the markets the token LEND and burn it to reduce monetary supply of the LEND token. 20% is reserved and directly redistributed to referees, in order to participate in the Referral Program, a referralCode to be used as an input of either deposit() or redeem() calls is needed.

You can receive a `referralCode` by completing [this](https://forms.gle/Va6dFgUmqwfn4bQL6) form and provide:
- Ethereum public address for funds reception

So basically, we need to set up a new public Ether wallet, which later might also be used to receive grants etc.

**Comment by Deepcryptodive** [comment]
Update: have requested code. Awaiting answer

**Comment by fodisi** [comment]
Implemented
